server/index.js

```diff
@@ -21,6 +23,14 @@ app.get('/products', (req, res) => {
       console.log('axios get products error', err);
     });
 });
+/* API for Questions */
+app.get('/questions', (req, res) => {
+  const cid = req.query.questionID;
```

Comment:
Move this block to questionsAPI.js and refactor so that this line can just be 
`app.get('/questions', questionsAPI.getQuestions)`

Reply:
Excellent idea. Refactoring now and will resubmit upon completion.